src/Angor/Avalonia/Angor.Wallet/Infrastructure/Impl/NetworkExtensions.cs

```diff
@@ -0,0 +1,38 @@
+using Angor.Wallet.Domain;
+using NBitcoin;
+
+namespace Angor.Wallet.Infrastructure.Impl;
+
+public static class NetworkExtensions
+{
+    public static Network ToNBitcoin(this BitcoinNetwork bitcoinNetwork)
```

Comment:
Not really crazy about exposing NBitcoin types like this, but if we have to...

Reply:
I understand your concern about exposing **NBitcoin** types, but this implementation is specifically designed for the infrastructure layer where we need this conversion to interact with the library.
The `ToNBitcoin` method is necessary to translate between our domain model (`BitcoinNetwork`) and NBitcoin's `Network` type, but it's encapsulated in this class and not exposed outside the infrastructure layer. 
There's no way for these conversions to reach the upper layers of the application, which maintains our domain isolation while allowing us to work with the **NBitcoin** library where needed.